src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -600,29 +604,56 @@ public Response showSettings(@Context HttpServletRequest request) throws IOExcep
 	@POST
 	@Path("/branches/delete")
```

Comment:
It would have to be escaped using javascript I think. But maybe that is some optimization for later ;)